dfget/core/api/supernode_api.go

```diff
@@ -115,6 +125,12 @@ func (api *supernodeAPI) ServiceDown(node string, taskID string, cid string) (
 
 	resp = new(types.BaseResponse)
 	e = api.get(url, resp)
+	if e != nil {
+		logrus.Infof("failed to send service down,err: %v", e)
+		return nil, e
+	} else if resp != nil && resp.Code != constants.CodeGetPeerDown {
```

Comment:
reduce the `else` please, like the above.

Reply:
Done